frontend/components/CompactNavigationBar.js

```diff
@@ -3,20 +3,19 @@ import { useRouter } from "next/router";
 import Link from "next/link";
 import Image from "next/image";
 import styles from "../styles/Navbar.module.css";
-
-// import menu icons
+import {
+  navigationBarContent,
+  aboutUsDropdownContent,
+  mediaDropdownContent,
```

Comment:
Remove `aboutUsDropDownContent` and `mediaDropdownContent`, seems like they aren't being used